charts/nri-prometheus/tests/labels_test.yaml

```diff
@@ -0,0 +1,58 @@
+suite: test object names
```

Comment:
As I already said, this is redundant to the test we do in the common library. I would remove them an use a `ci/test-values.yaml` to "simply" test that the chart installs.

Reply:
yes, ct test are already in place, i removed these tests.